LondonBritishLibrary/orient/BLorient672.xml

```diff
@@ -0,0 +1,370 @@
+                                        </item>
+                                        
+                                        <item xml:id="q28" n="27">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="229r" to="238v"/>
+                                        </item>
+                                        
+                                        <item xml:id="q29" n="28">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="239r" to="248v"/>
+                                        </item>
+                                        
+                                        <item xml:id="q30" n="29">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="249r" to="iv"/>
+                                            2, stub before 6
+                                            3, stub after 5
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="31 33">
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <!--vertical lines stop at the pricks, is it still A?-->
+                                    <ab type="ruling"> The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1700" notAfter="1749"/>
+                                <desc>Written in a good hand.</desc>
+                            </handNote>
+                        </handDesc>   
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#2r"/>
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of 
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
```

Comment:
Please note that if we say about the provenance etc. like this, we obscure the situation, as the book, as many other books were looted from other churches by Tewodros, and only after that they belonged for a while to Medhane Alam (before they were looted again).  Otherwise one might think Fasikawit, Mikael and Elyas donated the book to the Medhane Alam church.

Reply:
This is true for the entire BL collection; when we know where the MS came from before MA we put this too, but provenance is about the entire history, we write what we know... What is your suggestion?

Reply:
Since Fasikawit and Mikael have also been identified here https://github.com/BetaMasaheft/Manuscripts/pull/1456/commits/762d12382f4a720abdbba03e7282405d991ac06e can we assume that the MS came to MA from Qalāqǝl Māryām Ṣǝyon?

Reply:
Well, the story of MA collection is well-known, and the OwnershopNote for MA is also well recognizable. As it is now, it indicates that the book belongs (that means: still belong?) to MA. Previously a slightly different formula was used, somehow hinting to the fact that such a manuscript is one of the books brought by Tewodros II and donated to MA.

Reply:
Not from Qalaqel Maryam, but both books have smth in common, and the presence of the same persons should be clearly noted and made visible. QS-005 is itself a book  that could be brought to Qalaqel from elswere, and be one of more than 500 mss that the British took from Magdala and left in Tigray.

Reply:
> 

The different formula is used - here too - under provenance, the Ownership note as addition has simply the rendering of what the note says. I think this is correct as it is.